tinman/txgen.py

```diff
@@ -382,10 +383,35 @@ def build_actions(conf, silent=True):
     # setup processing time
     predicted_block_count += TRANSACTION_WITNESS_SETUP_PAD + (predicted_transaction_setup_seconds // STEEM_BLOCK_INTERVAL)
     
-    start_time = datetime.datetime.utcnow() - datetime.timedelta(seconds=predicted_block_count * STEEM_BLOCK_INTERVAL)
+    now = datetime.datetime.utcnow()
+    start_time = now - datetime.timedelta(seconds=predicted_block_count * STEEM_BLOCK_INTERVAL)
     miss_blocks = int((start_time - genesis_time).total_seconds()) // STEEM_BLOCK_INTERVAL
     miss_blocks = max(miss_blocks-1, 0)
+    origin_api = None
+    snapshot_head_block_num = None
+    snapshot_semver = None
+    
+    metadata = {
+      "txgen:semver": __version__,
+      "txgen:transactions_per_block": transactions_per_block,
```

Comment:
`recommend:transactions_per_block` - thinking about our conversations about making recommendations explicit

Reply:
Does that mean the name is wrong?  Or is the issue that this value isn't being used by anything?

Reply:
name preference

Reply:
OIC, you're thinking of it as a declaration of generation settings, rather than a recommendation. SG.